source/numerics/vector_tools_integrate_difference.inst.in

```diff
@@ -132,12 +132,660 @@ for (VEC : VECTOR_TYPES; deal_II_dimension : DIMENSIONS;
                            deal_II_space_dimension>(
         const hp::DoFHandler<deal_II_dimension, deal_II_space_dimension> &,
         const VEC &,
+        const Function<deal_II_space_dimension, typename VEC::value_type> &,
+        Vector<float> &,
+        const hp::QCollection<deal_II_dimension> &,
+        const NormType &,
+        const Function<deal_II_space_dimension> *,
+        const double);
+
+    \}
+#endif
+  }
+
+// All the functions in this for loop are deprecated
+for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS)
+  {
```

Comment:
I had to write a long list of instantiations for the deprecated functions. I can not loop over `VECTOR_TYPES` because I run in conflict with the new function.

Reply:
Hmmm... Yes, that is annoying. Thanks for doing it anyway. Let me see if I can find something better here. 

Apart from that, I am fine with the pull request.

Reply:
Sounds good, let me know if you find a better approach

Reply:
I didn't. Let's just move forward. Maybe we find a better way later.

Reply:
Sounds good!